.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Create a report to help us improve
+title: "Bug Report"
+labels: [bug]
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to file a Bug report!
+  - type: textarea
+    attributes:
+      label: Describe the bug.
+      description: "Enter a clear and concise description of the bug."
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the steps to reproduce the behavior.
+      description: "Clearly and concisely describe the steps to ensure that it can be reproduced and potentially fixed."
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+  - type: textarea
+    attributes:
+      label: Expected behavior.
+      description: "Enter a clear and concise description of what you expected to happen."
```

Comment:
Hi @jhdalek55  for additional context, I think it was referring to the reason why they were sending the bug report -- they expected X, but got Y. So they're sending the bug report, because X did not happen and they want that fixed. Let me know if you'd like to add/edit anything with this additional context or if you'd like me to commit your suggested change.

Reply:
Ok. Understood. I'll review it again later.

Reply:
Fine, then. I remove my objection.